### ISSUE TITLE
Background PopupContentViewController in Dark Mode

### ISSUE THREAD

**Issue by balabon7** [iDevelopper/PBPopupController#28]
Hello 👋

When I switch the device in a Dark Mode, the PopupContentViewController background is automatically darkened.

Please tell me where in the library code you can change the darkening level or remove the dimming?

(At least in debugging mode)

![IMG_9707](https://github.com/iDevelopper/PBPopupController/assets/42384032/af522f03-5a34-4f28-ab3e-32197f43e319)
![IMG_9708](https://github.com/iDevelopper/PBPopupController/assets/42384032/dede0fd2-edb4-42f6-8b75-06443e321fcf)

**Comment by iDevelopper** [comment]
Hello,

You can have a look at the overrideUserInterfaceStyle property (not used in the example project) if you want to force the view or all views (window) to always adopt a light or dark interface style.

Otherwise, since iOS 13 the UI Element Colors objects adapt automatically to Dark Mode changes when you use the provided UIColor object (https://developer.apple.com/documentation/uikit/uicolor/ui_element_colors).

In the example, in PopupContentViewController.swift, you can see:
```swift
        if #available(iOS 13.0, *) {
            self.view.backgroundColor = UIColor.secondarySystemBackground
        }
```
This color automatically changes when the dark mode changes.

You can change the color by the UI Element Color (secondarySystemBackground) by the standard color of your choice (https://developer.apple.com/documentation/uikit/uicolor/standard_colors).

**Comment by balabon7** [comment]
Thank you very much for your help, dismantled with a color change 👍
